packages/tools/pluggable-widgets-tools/configs/rollup.config.js

```diff
@@ -90,7 +90,8 @@ export default async args => {
                 external: nativeExternal,
                 plugins: [
                     replace({
-                        "Platform.OS": `"${os}"`
+                        "Platform.OS": `"${os}"`,
+                        preventAssignment: true
```

Comment:
What does this do? 😮

Reply:
Avoid that if constants match the replace statement followed by '=' sign